plugins/system-update/PageComponent.qml

```diff
@@ -172,6 +172,13 @@ ItemPage {
                 }
             }
 
+            ListItem.SingleValue {
+                objectName: "updateApps"
+                text: i18n.tr("Update apps in the OpenStore")
```

Comment:
Looks OK to me, but I wonder if "Check for app updates in the OpenStore" is better? (I'll leave it to you, as I'm not a native speaker :-) )